app/scripts/pages/navigator.jsx

```diff
@@ -15,11 +15,10 @@ import { UrlGen, Path } from '../util/urlEnum';
 @observer
 class Navigator extends React.Component {
     componentDidMount() {
-        const { leftMenuDrawer, openTagManager, selectedEntity } = mainStore;
```

Comment:
Looks like `openTagManager ` and `selectedEntity ` are never used in this method?